### ISSUE TITLE
Roundtrip failure (problematic hash key?) [rt.cpan.org #32875]

### ISSUE THREAD

**Issue by ingydotnet** [ingydotnet/yaml-pm#88]
https://rt.cpan.org/Ticket/Display.html?id=32875

```
The attached script fails with YAML.pm with the message:

YAML Error: Couldn't parse single line value
   Code: YAML_PARSE_ERR_SINGLE_LINE
   Line: 2
   Document: 1
 at /usr/local/lib/perl5/site_perl/5.8.8/YAML.pm line 33
Exitcode 255

With YAML::Syck it works as expected.

Regards,
    Slaven
```

**Comment by perlpunk** [comment]
Related to #36 and #48

**Comment by perlpunk** [comment]
fixed in 1.20
